package.json

```diff
@@ -1,13 +1,17 @@
 {
   "name": "solidity-coverage",
-  "version": "0.4.0",
+  "version": "0.4.1",
   "description": "",
```

Comment:
Let's leave this at `0.4.0` and let npm do it when we run `npm version patch` and publish.

Reply:
ok